File: renter/check_db.php
<?php
require_once "../db.php";
echo "--- payment_notifications ---\n";
$q = mysqli_query($conn, "SELECT id, amount, sys_tx_id, transaction_id, status FROM payment_notifications ORDER BY id DESC LIMIT 5");
while($r = mysqli_fetch_assoc($q)) {
    print_r($r);
}

echo "\n--- payments ---\n";
$q2 = mysqli_query($conn, "SELECT id, sys_tx_id, transaction_id FROM payments ORDER BY id DESC LIMIT 5");
while($r = mysqli_fetch_assoc($q2)) {
    print_r($r);
}

File: renter/check_db.py
import mysql.connector

conn = mysql.connector.connect(
    host="localhost",
    user="root",
    password="",
    database="renter_system"
)

cursor = conn.cursor(dictionary=True)

print("--- payment_notifications ---")
cursor.execute("SELECT id, sys_tx_id, transaction_id, status FROM payment_notifications ORDER BY id DESC LIMIT 5")
for row in cursor.fetchall():
    print(row)

print("\n--- payments ---")
cursor.execute("SELECT id, sys_tx_id, transaction_id FROM payments ORDER BY id DESC LIMIT 5")
for row in cursor.fetchall():
    print(row)

conn.close()
